### ISSUE TITLE
Isn't the message in the popup clear enough?

### ISSUE THREAD

**Issue by softweaver** [eclipse/wildwebdeveloper#511]
Isn't the message in the popup clear enough?
```
Could not find node.js. This will result in editors missing key features.
Please make sure node.js is installed and that your PATH environment variable contains the location to the `node` executable.
```

_Originally posted by @mickaelistria in https://github.com/eclipse/wildwebdeveloper/issues/264#issuecomment-534689498_

Clear?

Clearly a bug.

Clearly obnoxious.

Clearly slowing down normal usage of Eclipse.

I don't want to have node.js installed.

I don't want to have node.js installed anywhere on my machine. Beyond the bare minimum that I consciously select, I don't want random native executables, especially language interpreter native executables, on my dev machine for security reasons. And for other reasons, but security concerns should be more than sufficient.

I do not use node.js -- I do not develop for it, or use tools based on it, or want it to be part of my eclipse tool stack. 

Based on the past 20 years of working with EJBOSS, JBoss, Wildfly, & Quarkus, I'm pretty confident that I won't want node.js physically installed on my local machine in the coming 20 years.

A modal popup pestering me to install node.js, and interfering with and slowing down the most basic editing operations is leading to some of the Worst User Experience in the ~17 years I've been using Eclipse.

At minimum, the modal should have a button leading to a quick & simple way of uninstalling whatever plugin(s) are demanding node.js.

People choose Java-based tools for a wide variety of reasons, and trustworthiness and performance and security and maturity and reliability are key among them. Eclipse may be applied to edit an unlimited array of polyglot source code, no problem there, as long as those runtimes are not bundled and are completely optional -- but when polyglotism creeps in to the operation of the tool itself, and then starts interfering with basic usability & robustness -- we have to draw a line.

**Comment by mickaelistria** [comment]
I read it that the message is clear enough, and you just don't want to have node.js installed. So it's not a bug in Wild Web Developer.
You're free to not use Wild Web Developer if you don't want to install the Node.js embedder nor have node.js installed in your machine. You can totally uninstall it if it suits your better.

**Comment by softweaver** [comment]
After dozens of hours of slowing down my work with inexplicable unresponsiveness, sprinkled with dozens of irrelevant popups that I dismissed and ignored, I eventually realized that this popup demanding node.js was connected to the slowdowns. Then it took me several hours to research and find the cause of why this is happening.

Thousands of less-patient devs wont ever find out what is going on.

So the way that it is bundled is at minimum a UX issue for all of Eclipse.

Keep in mind that I downloaded the vanilla JEE version of Eclipse. Approximately just as I have 20+ times before. I did not ask for "Wild Web Developer", I did not install it. It is bundled without warning.

What are the steps to reliably uninstall it -- together with removing all of any hair that it might have brought with it?
